code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drugs.dm

```diff
@@ -108,8 +110,13 @@
 /decl/reagent/mindbreaker/affect_blood(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
 	M.hallucination = max(M.hallucination, 100)
 	M.add_chemical_effect(CE_HALLUCINATE, 2)
-	if(prob(10))
-		M.add_chemical_effect(CE_NEUROTOXIC, 5*removed)
+	M.add_chemical_effect(CE_NEUROTOXIC, 5*removed)
+
+/decl/reagent/mindbreaker/overdose(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
+	M.add_chemical_effect(CE_NEUROTOXIC, 5*removed)
```

Comment:
This might be an oversight, but according to this bit of code for Mindbreaker, it will do a good amount of brain damage by default, even when not overdosing. Is that intended?
Otherwise I'd recommend being careful since we do have Mindbreaker as part of the scrubber discharge thing.

Reply:
The non-OD number was an oversight, yeah, especially with scrubbers in mind. It's be toned down, but still lethal when ODing.